sigs.yaml

```diff
@@ -111,17 +108,49 @@ sigs:
             github: eddev
             company: Red Hat
       subprojects:
-        - name: nmstate
+        - name: kubevirt-network
+          leads:
+          - name: "Edward Haas"
+            github: eddev
+            company: Red Hat
           owners:
-            - https://raw.githubusercontent.com/nmstate/kubernetes-nmstate/main/OWNERS
-        - name: k8snetworkplumbingwg
+            - https://raw.githubusercontent.com/kubevirt/kubevirt/main/OWNERS_ALIASES
```

Comment:
nit: It may be too specific, but how about naming the specialized OWNERS files referring towards `sig-network-*` groups?

(Could be done in a followup - I'd event think about automatic generation here)

Reply:
OWNERS files are many times located at different folders and these move a lot during refactoring.
It will be uncomfortable to track this and make changes to this meta file every time the codebase changes (even if it is automated).

It will also be correct only for the `main` branch but not for the stable releases.
I would suggest to leave it open like it is now, but we could provide tooling (script?) to identify all packages that are owned by a specific SIG, unrelated to this meta file.

Reply:
All fair points - I agree that the most pragmatic approach is to declare OWNERship by targeting the root file.